Packs/VersaDirector/Integrations/VersaDirector/test_data/input_data.py

```diff
@@ -0,0 +1,325 @@
+            "application": {
+                "predefined-application-list": "[]",
+                "user-defined-application-list": "[]",
+            },
+            "url-category": {"user-defined": "[]"},
+            "url-reputation": {"predefined": "[]"},
+            "ttl": {},
+        },
+        "set": {
+            "lef": {"event": "never", "profile-default": "true", "rate-limit": 10},
+            "action": "",
+            "tcp-optimization": {},
+            "forwarding-profile": "",
+            "nexthop-address": "",
+            "routing-instance": "",
+        },
+        "monitor": {},
+    }
+}
+appliance_sdwan_policy_rule_command_readable_output = "Command run successfully.\nRequest Body:\n\n" + str(
+    appliance_sdwan_policy_rule_json
+)
+
+
+# Test function: template_address_object_create_command, template_address_object_edit_command
+template_address_object_json = {
+    "address": {
+        "name": "object_name",
+        "description": "",
+        "tag": "[]",
+        "address_object_type": "object_value",
+    }
+}
+template_address_object_command_readable_output = "Command run successfully.\nRequest Body:\n\n" + str(
+    template_address_object_json
+)
+
+
+# Test function: appliance_address_object_create_command, appliance_address_object_edit_command
+appliance_address_object_json = {
+    "address": {
+        "name": "object_name",
+        "description": "",
+        "tag": "[]",
+        "address_object_type": "object_value",
+    }
+}
+appliance_address_object_command_readable_output = "Command run successfully.\nRequest Body:\n\n" + str(
+    appliance_address_object_json
+)
+
+
+# HEADING: --------------- HELPER FUNCTIONS TESTS ARGUMENTS ---------------
+
+
+# Test function: test_set_organization
+# Arguments: (organization_form_args,organization_form_params, expected_output)
+case_organization_form_args = ("args", None, "args")
+case_organization_form_params = (None, "params", "params")
+case_organization_form_both = ("args", "params", "args")
+set_organization_args = [
+    case_organization_form_args,
+    case_organization_form_params,
+    case_organization_form_both,
+]
+
+
+# Test function: test_set_offset, test_set_offset_fail
+# Arguments: (page, page_size, expected_output)
+case_page_none_and_page_size_zero = (None, 0, 0)
+case_page_none_and_page_size_value = (None, 1, 1)
+case_page_value_and_page_size_none = (1, None, 1)
+case_page_none_and_page_size_none = (None, None, None)
+case_page_value_and_page_size_value = (2, 2, 4)
+
+page_negative_and_page_size_positive = (-1, 1, "page or page_size arguments are invalid.")
+page_positive_and_page_size_negative = (1, -1, "page or page_size arguments are invalid.")
+page_negative_and_page_size_negative = (-1, -1, "page or page_size arguments are invalid.")
+
+set_offset_args = [
+    case_page_none_and_page_size_zero,
+    case_page_none_and_page_size_value,
+    case_page_none_and_page_size_none,
+    case_page_none_and_page_size_none,
```

Comment:
```suggestion
    case_page_value_and_page_size_none,
```
You deleted the wrong line